packages/stores/types.ts

```diff
@@ -577,7 +604,7 @@ type TTradersHubStore = {
             landing_company_short?: 'bvi' | 'labuan' | 'svg' | 'vanuatu' | 'maltainvest';
             platform?: string;
             description?: string;
-            market_type?: 'all' | 'financial' | 'synthetic';
+            market_type?: 'all' | 'financial' | 'gaming';
```

Comment:
@thisyahlen-deriv why did you change this type?

Reply:
Because there is no such thing as synthetic market_type in the response. Its only gaming. We are changing it to synthetic somewhere in the code.